src/static.ts

```diff
@@ -126,6 +126,10 @@ export interface MynahUIDataModel {
    * Tab content header details, only visibile when showTabHeaderDetails is set to 'true'
    */
   tabHeaderDetails?: TabHeaderDetails | null;
+  /**
+   * The navigation index for the chat prompts list for the up/down arrow key navigation
+   */
+  navigationIndexChatPrompts?: number;
```

Comment:
Maybe change to `upDownKeyNavigationIndex` for consistency with `UP_DOWN_ARROW_KEY_PRESS` naming

Reply:
> We should already have them in the keys list.

sorry I didnt get this, not sure I know what is the key list and what you mean with 'them'?